src/service/member-api-adapter/tests/test-member-api-service.js

```diff
@@ -105,11 +105,25 @@ describe('Member API Service', function () {
         expect(req.requestBody).to.eq(JSON.stringify({ active: active }));
     };
 
+    var testPatchUsersActiveField = function (active) {
```

Comment:
nitpick, call it `testPatchMultiUserActiveField ` instead; had to read twice to realize `testPatchUsersActiveField` wasn't the same as `testPatchUserActiveField`